### PR TITLE
add sign in screen and home screen to Storyboard

### PR DESCRIPTION
**Pull request by MohamedSaiko**
add sign in screen using storyboard

### CODE REVIEW

**Comment by akabel92** [comment]
Please add descriptive name for the PR title

File: Food Advisor/Home/HomeViewController.swift [hunk 1]
```diff
@@ -0,0 +1,29 @@
+//
+//  HomeViewController.swift
+//  Food Advisor
+//
+//  Created by Mohamed Sayed on 07.04.23.
+//
+
+import UIKit
+
+class HomeViewController: UIViewController {
```
Comment (akabel92) [inline Food Advisor/Home/HomeViewController.swift:10]: Why do you want to add empty class?

File: Food Advisor/Food_AdvisorApp.swift [hunk 2]
```diff
@@ -12,8 +12,8 @@ struct Food_AdvisorApp: App {
     var body: some Scene {
         WindowGroup {
             //ContentView()
-            //HomeScreen()
-            SignInScreen()
+            HomeScreen()
+            //SignInScreen()
```
[the remark this answers is not in the document]
Comment (MohamedSaiko) [reply Food Advisor/Food_AdvisorApp.swift:16]: no, not with storyboard i will delete them

File: Food Advisor/Home/HomeViewController.swift (hunk 1 above)
Comment (MohamedSaiko) [reply Food Advisor/Home/HomeViewController.swift:10]: I will use it later, I should have ignored it before committing.

File: Food Advisor/Home/HomeViewController.swift [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Do any additional setup after loading the view.
+    }
+    
+
+    /*
+    // MARK: - Navigation
```
[the remark this answers is not in the document]
Comment (MohamedSaiko) [reply Food Advisor/Home/HomeViewController.swift:20]: Ok, I will

**Comment by MohamedSaiko** [comment]
@akabel92 
Hello Ali,
i'm waiting for your feedback on the last commit (add home screen).

**Review by akabel92 — APPROVED** [review]

**Comment by akabel92** [comment]
> @akabel92 Hello Ali, i'm waiting for your feedback on the last commit (add home screen).

approved, feel free to merge it :)

**Pull request merged**